in_toto/in_toto_mock.py

```diff
@@ -100,7 +80,15 @@ def main():
   # TRY out in-toto-run, with fewer command lines args and max. user feedback.
   log.setLevel(logging.INFO)
 
-  in_toto_mock(args.name, args.link_cmd)
+  try:
+    in_toto.runlib.in_toto_mock(args.name, args.link_cmd)
+
+  except Exception as e:
+    log.error("in toto mock - {}".format(e))
```

Comment:
Why is it written as `in toto mock` and not `in_toto_mock.py` or `in-toto mock`?

Reply:
Good question. I'll make uniform with `log.error`s in other command line tools.